test/functional/test_framework/mininode.py

```diff
@@ -130,7 +134,7 @@ def __init__(self):
     def is_connected(self):
         return self._transport is not None
 
-    def peer_connect(self, dstaddr, dstport, *, net, timeout_factor):
+    def peer_connect_helper(self, dstaddr, dstport, net, timeout_factor):
```

Comment:
nit: Any reason to remove the star? I think keyword args that are simply passed on in python are denoted by `**kwargs` in intermediate function calls in the stack, not by explicitly listing them each. Though, I am not a python expert, so feel free to ignore.

Reply:
Yeah, the `*` forces subsequent params to be specified by keyword not position

Reply:
yup, thats why I removed the `*` from the helper. 

also note that the `*` is still present in the `peer_connect` function, just git is showing the diff since I added the helper. I don't fully understand what's going on here with the `*` on the actual functions though (`peer_connect`, `peer_accept_connection`), so please let me know if I'm missing something / should dig in more.